### PR TITLE
Fix docker build

### PR DESCRIPTION
**Pull request by AttackingOrDefending**
## Type of pull request:
- [x] Bug fix
- [ ] Feature
- [ ] Other

## Description:

Hopefully fixed the error. Changed `DOCKER_ORGANIZATION` to `lichessbotdevs` and added `GHCR_DOCKER_ORGANIZATION` with a value of `lichess-bot-devs`. DockerHub didn't let me use hyphens in the username, so the username is `lichessbotdevs`. I can test whether this works on a new branch in this repo if you want before merging. 

## Related Issues:

#949 

## Checklist:

- [x] I have read and followed the [contribution guidelines](/docs/CONTRIBUTING.md).
- [x] I have added necessary documentation (if applicable).
- [x] The changes pass all existing tests.

### CODE REVIEW

**Comment by AttackingOrDefending** [comment]
We can also remove all the `VARS.variable` and change them with the value, so that they are easier to debug.

**Comment by MarkZH** [comment]
Do the test and merge if it works.

**Review by MarkZH — APPROVED** [review]

**Comment by AttackingOrDefending** [comment]
In the GitHub container registry we have to use `docker pull ghcr.io/lichess-bot-devs/lichess-bot:latest` and on docker hub `docker pull lichessbotdevs/lichess-bot:latest`. We should probably mention that on dockerhub there are no hyphens, while there are on ghcr. I don't know how lines mentioning `lichessbotdevs` (e.g. `FROM lichessbotdevs/lichess-bot:alpine`) have to change for ghcr. I assume just adding hyphens won't be enough. @fathzer Can you help with how each mention of `lichessbotdevs` must change for ghcr?

File: wiki/How-to-use-the-Docker-image.md
Comment (fathzer) [inline wiki/How-to-use-the-Docker-image.md:1]: You should replace `https://hub.docker.com/repository/docker/lichess-bot-devs/lichess-bot` by `https://hub.docker.com/repository/docker/lichessbotdevs/lichess-bot` in line 33

**Comment by fathzer** [comment]
It seems your Github package is not public ; When I'm logged out from Github, `https://github.com/fathzer/lichess-bot/pkgs/container/lichess-bot` shows my test repo, but `https://github.com/lichess-bot-devs/lichess-bot/pkgs/container/lichess-bot` results in a 404 error page.

Here is an example of how to pull an image from Github: `docker pull ghcr.io/fathzer/lichess-bot`. In the same way, in the run command, the image name should be prefixed by `ghcr.io/`.

**Comment by AttackingOrDefending** [comment]
> It seems your Github package is not public

Thanks. Fixed it.

@MarkZH Look at the docs, and see if you agree with the changes. I don't have any other changes to make.

File: wiki/How-to-use-the-Docker-image.md [hunk 1]
```diff
@@ -1,18 +1,26 @@
 # How to launch your bot
+## Download the docker image
+The images are available through [Docker hub](https://hub.docker.com/r/lichessbotdevs/lichess-bot) and [Github container registry](https://github.com/lichess-bot-devs/lichess-bot/pkgs/container/lichess-bot).
+Run `docker pull lichessbotdevs/lichess-bot` to download from docker hub, or run `docker pull ghcr.io/fathzer/lichess-bot` if you want to download it from the Github container registry.
```
Comment (MarkZH) [inline wiki/How-to-use-the-Docker-image.md:4]: Is `docker pull ghcr.io/fathzer/lichess-bot` correct?

Comment (AttackingOrDefending) [reply wiki/How-to-use-the-Docker-image.md]: Fixed.

**Comment by AttackingOrDefending** [comment]
I added aan action that pushes the docker docs to docker hub, and made the workflow first build the alpine image and then the main image because ghcr shows how to install the image that was pushed last.

**Review by MarkZH — APPROVED** [review]

**Comment by MarkZH** [comment]
Merge if ready.

**Pull request merged**